code/game/objects/structures/tank_dispenser.dm

```diff
@@ -34,6 +35,22 @@
 		if(5 to TANK_DISPENSER_CAPACITY)
 			. += "plasma-5"
 
+/obj/structure/tank_dispenser/attack_hand(mob/living/user, list/modifiers)
+	. = ..()
+	if (plasmatanks)
+		dispense(/obj/item/tank/internals/plasma, user)
+		plasmatanks--
+		update_appearance()
+		return
+
+/obj/structure/tank_dispenser/attack_hand_secondary(mob/user, list/modifiers)
+	. = ..()
+	if (oxygentanks)
```

Comment:
early return

Reply:
`attack_hand` will be called if I won't return in `attack_hand_secondary`

Reply:
ok so please early return